configure

```diff
@@ -100,6 +101,40 @@ while [ "$TF_NEED_HDFS" == "" ]; do
   esac
 done
 
+while [ "$TF_NEED_LIBJPEG_TURBO" == "" ]; do
```

Comment:
There might be things @jhseu hadn't considered.

World will not end if we depend on nasm. It's 0.8MB and can be built very easily with Bazel. https://gist.github.com/jart/68b1fd09d48197850981d45e85758936 (Although that BUILD file would need a genrule to create config.h and the copts would need to be tuned for MSVC. We do not need to depend on autotools.) We already download and build others tools from scratch, e.g. [SWIG](https://github.com/tensorflow/tensorflow/blob/master/third_party/swig.BUILD). It adds three seconds to build time on my workstation. That's better IMHO than increasing the cognitive burden on users with configuration questions and action items.

However building libjpeg-turbo inside Bazel is not trivial. We have a build file that does it internally. It's a very complete build file that appears to support x86_64, x86, armv7a, armv8a, Android, GCC, Clang, and MSVC. I might be able to copy this file onto GitHub. Thoughts @gunan and @wicke?